### ISSUE TITLE
US Govt 508 Compliance Issues

### ISSUE THREAD

**Issue by gyellin** [gephi/gephi#937]
Issue #1: When the “About” pop up window appears, JAWS states “About, use Tab key to navigate” without reading any of the textual information in the pop up window. 

Expected result:  All textual information is expected to be made available for JAWS to announce.

Issue #2: When navigating through the toolbar, the following links are unavailable for JAWS to announce:  File, “Workspace”, “View”, “Tools”, “Window”, “Plugins”, and “Help”.

Expected result:  All user interface elements are expected to be made available for JAWS to announce.

Issue #3: When attempting to navigate through the screen by using the Tab key all of the user interface elements outside of the new project window and the toolbar are not accessible via the key board.  

This is a medium defect because the user is given the option to create key short cuts.  To access this option inside of the “Tools” tab select options and navigate to keymap.  

Expected results:  All user interface elements are expected to be accessible via the keyboard.

Issue #4: The text box fields and the associated labels are unavailable for JAWS to announce.  As example the first text box field should be announced as, “Name: edit type in text, Project 0”.  

Expected result:  All text box fields and the associated labels are expected to be made available for JAWS to announce.  In this instance the first text box should be made available for JAWS to announce, “Name: edit type in text, Project 0”.
